client/components/CalendarDays.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import moment from 'moment';
+
+const CalendarDays = () => {
+  const weekdaysshort = moment.weekdaysShort();
```

Comment:
Camel-case variable names
```suggestion
  const weekdaysShort = moment.weekdaysShort();
```

Reply:
also line 7

Reply:
variable naming correct.